packages/calling/src/Voicemail/Voicemail.ts

```diff
@@ -115,12 +123,12 @@ export class Voicemail extends Eventing<VoicemailEventTypes> implements IVoicema
   }
 
   /**
-   * Call voicemail class to fetch the voicemail lists.
+   * Retrieves a list of voicemails with optional pagination and sorting options.
    *
-   * @param sort - Sort voicemail list (ASC | DESC). TODO: Once we start implementing sorting.
    * @param offset - Number of records to skip.
-   * @param offsetLimit - Number of voicemail list to fetch from the offset.
-   * @param refresh - Refresh the list of voicemails from backend.
+   * @param offsetLimit - The limit on the number of voicemails to retrieve from the offset.
+   * @param sort - Sort voicemail list (ASC | DESC). TODO: Once we start implementing sorting.
+   * @param refresh - Set to `true` to force a refresh of voicemail data from backend (optional).
    * @returns Promise.
```

Comment:
Correct this...@return Promise doesn't tell anyhting

Reply:
Removed @returns